zerver/openapi/zulip.yaml

```diff
@@ -5851,6 +5851,51 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/JsonSuccess"
+    patch:
+      operationId: update_linkifier
+      tags: ["server_and_organizations"]
+      description: |
+        Update [linkifiers](/help/add-a-custom-linkification-filter), regular
+        expression patterns that are automatically linkified when they appear
+        in messages and topics.
+
+        `PATCH {{ api_url }}/v1/realm/filters/{filter_id}`
+      parameters:
+        - name: filter_id
+          in: path
+          description: |
+            The ID of the filter that you want to update.
```

Comment:
We use the term "linkifier" rather than "filter" in all user-facing docs, and probably this text should be included.

(More medium-term, we're going to eventually rename "filters" to "linkifiers" in the API; it's just a bit of an annoying transition because of mobile app compatibility issues)

Reply:
I have renamed this to `linkifier` and have also changed the same over [here](https://github.com/zulip/zulip/pull/17045/files#diff-7d09c06f7ab16ed46eeff8eb6e1a342f75fc9f3c780c6f10643303145316581eR5895).